### ISSUE TITLE
Gaussian Splatting to fail

### ISSUE THREAD

**Issue by Amos1109** [ByteDance-Seed/Depth-Anything-3#200]
I tried to apply Gaussian Splatting to some video frames, but the generated results looked like this, whereas the final MP4 video output had no issues. Why might this be happening? I used DA3NESTED-GIANT-LARGE-1.1.

<img width="2283" height="1275" alt="Image" src="https://github.com/user-attachments/assets/91a1448a-9db3-4ebb-a367-ffbce647f791" />

<img width="1497" height="448" alt="Image" src="https://github.com/user-attachments/assets/78300040-7e1e-45e7-8ddf-6fb5d5a53384" />

This is my code.

```
import glob, os, torch
from depth_anything_3.api import DepthAnything3
device = torch.device("cuda:3")
model = DepthAnything3.from_pretrained("/data23/hong_yuting/Depth-Anything-3/da3_streaming/weights")
model = model.to(device=device)
example_path = "/data23/hong_yuting/Depth-Anything-3/office_images"
images = sorted(glob.glob(os.path.join(example_path, "*.png")))

output_dir = "outputs/demo"
export_format = "gs_ply"

model.inference(
    images,
    extrinsics=None,
    intrinsics=None,
    infer_gs=True,
    export_dir=output_dir,
    export_format=export_format,
    export_feat_layers=[],
)
```

**Comment by Amos1109** [comment]
```
import glob
import os
import torch
from depth_anything_3.api import DepthAnything3
from depth_anything_3.utils.gsply_helpers import save_gaussian_ply

# 配置参数
model_path = "/data23/hong_yuting/Depth-Anything-3/da3_streaming/weights"  # 模型路径
image_path = "/data23/hong_yuting/Depth-Anything-3/da3_streaming/images_1222_100_new_user/"  # 图像目录路径
device = torch.device("cuda:3" if torch.cuda.is_available() else "cpu")  # 设备

print("\n[1/4] 加载模型...")
model = DepthAnything3.from_pretrained(model_path)
model = model.to(device=device)
print(f"✅ 模型加载成功: {model_path}")

# 2. 加载图像
print("\n[2/4] 加载图像...")
all_images = sorted(glob.glob(os.path.join(image_path, "*.jpg")))
# 建议：至少 5-10 张图像，可以调整这个数字
num_images = min(100, len(all_images))  # 使用前10张（或全部如果不足10张）
images = all_images[:num_images]
print(f"✅ 加载了 {len(images)} 张图像（共 {len(all_images)} 张可用）")
for i, img in enumerate(images):
    print(f"  - 图像 {i}: {img.split('/')[-1]}")

# 3. 推理（生成 GS）
print("\n[3/4] 运行前馈 GS 推理...")
print("⏳ 正在预测深度 + 姿态 + 高斯参数...")

try:
    prediction = model.inference(
        images,
        infer_gs=True,  # 🔥 关键参数：启用 GS 预测
        # infer_depth=True,  # 默认开启
        # infer_camera=True, # 默认开启
    )
    print("✅ 推理完成")
    
    # 从 prediction 中提取 gaussians 和 depth
    gs_world = prediction.gaussians
    pred_depth = torch.from_numpy(prediction.depth).unsqueeze(-1).to(gs_world.means)  # v h w 1
    
    # 创建输出目录
    improved_output_dir = "outputs/gs_test"
    os.makedirs(os.path.join(improved_output_dir, "gs_ply"), exist_ok=True)
    
    # 保存高斯点云
    save_gaussian_ply(
        gaussians=gs_world,
        save_path=os.path.join(improved_output_dir, "gs_ply/0001_no_filter.ply"),
        ctx_depth=pred_depth,
        shift_and_scale=True,  # 启用坐标归一化
        save_sh_dc_only=True,
        gs_views_interval=1,
        inv_opacity=True,
        prune_by_depth_percent=1.0,  # 🔥 完全禁用深度过滤
        prune_border_gs=False,  # 🔥 禁用边界过滤
        match_3dgs_mcmc_dev=False,
    )
    print("✅ 高斯点云保存完成")
except Exception as e:
    print(f"❌ 错误: {e}")
    import traceback
    traceback.print_exc()
```

is also not work.

**Comment by devotetime** [comment]
hi amos！I also encountered the same problem. Could you tell me if you have solved it? I suspect that the parameters of the Gaussian sphere when exporting the ply file might be the cause of the issue.

**Comment by 20713** [comment]
> 嗨，Amos！我也遇到了同样的问题。请问您解决了吗？我怀疑导出ply文件时高斯球的参数可能是问题所在。

请问您解决了吗？

**Comment by suzhaoqi** [comment]
You can try to use some viewers designed for viewing 3DGS like https://playcanvas.com/supersplat/editor, instead of directly opening the 3DGS ply in meshlab.
